### ISSUE TITLE
Set up tests for driver and library

### ISSUE THREAD

**Issue by 23ars** [23ars/linux_gpio_driver#3]

**Comment by prernapaliwal** [comment]
Have you developed an application for testing this driver like setting, reading the valur of a certain GPIO pins?

**Comment by 23ars** [comment]
@prernapaliwal  yes, there is an example of code in [Wiki/Use](https://github.com/23ars/linux_gpio_driver/wiki/Use)
